config/locales/api/en.yml

```diff
@@ -470,4 +473,25 @@ en:
     oral_cancer: "Oral cancer"
     breast_cancer: "Breast cancer"
     cervical_cancer: "Cervical cancer"
-
+    # Ethiopia
+    et_total_bp: "Total BP screening done"
```

Comment:
There is varying text per country. Should we namespace them under a key (i.e. `et.total_bp`) or simply prefix the country code (i.e. `et_total_bp`)?
I think namespacing would be neater. Thoughts?

Reply:
I have put them under comments & without any prefix or namespace as same string can be reused in future.